{"Functions": {"ClosedByInterruptException()": {"Returns": [], "Parameters": [], "Throws": [], "SeeAlso": [], "Permissions": [], "Description": "Constructs an instance of this class.", "history": "Added in API level 1", "FullName": "public ClosedByInterruptException ()"}}, "Inheritance": ["@B_java#lang#Throwable_E@", "@B_java#lang#Exception_E@", "@B_java#io#IOException_E@", "@B_java#nio#channels#ClosedChannelException_E@", "@B_java#nio#channels#AsynchronousCloseException_E@"], "ClassName": "java.nio.channels.ClosedByInterruptException", "ClassDesc": "Checked exception received by a thread when another thread interrupts it while it is blocked in an I/O operation upon a channel. Before this exception is thrown the channel will have been closed and the interrupt status of the previously-blocked thread will have been set."}